updating/updating_a_cluster/control-plane-only-update.adoc

```diff
@@ -64,3 +64,10 @@ include::modules/updating-control-plane-only-layered-products.adoc[leveloffset=+
 * xref:../../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators]
 * xref:../../updating/updating_a_cluster/control-plane-only-update.adoc#updating-control-plane-only-update-console_control-plane-only-update[Performing a Control Plane Only update using the web console]
 * xref:../../updating/updating_a_cluster/control-plane-only-update.adoc#updating-control-plane-only-update-cli_control-plane-only-update[Performing a Control Plane Only update using the CLI]
+
+
+[role="_additional-resources"]
+[id="additional-resources_control-plane-only-updates"]
+== Additional resources
```

Comment:
This doesn't look right to me. I think you just want to add this to the Additional resources list on line 68, especially given the theme of that list.

Reply:
That does make sense. Updated as per the suggestion. Thanks!